support-frontend/assets/components/onboarding/sections/summary.tsx

```diff
@@ -0,0 +1,263 @@
+	existingUserSignedIn: {
+		title: "You're ready to go",
+		description:
+			"Find out what's included in your All-access digital subscription.",
+	},
+	userSignedIn: {
+		title: 'You’re signed in',
+		description: 'You can now explore your exclusive benefits.',
+	},
+	userRegistered: {
+		title: 'Welcome to the Guardian',
+		description:
+			'Your account is set up and ready to go. Now you can explore your exclusive benefits.',
+	},
+};
+
+export function OnboardingSummarySuccessfulSignIn({
+	handleStepNavigation,
+	userState,
+}: {
+	handleStepNavigation: HandleStepNavigationFunction;
+	userState: CurrentUserState;
+}) {
+	const [switchNewsletterEnabled, setSwitchNewsletterEnabled] = useState(true);
+
+	return (
+		<Stack space={2}>
+			<h1 css={headings}>{onboardingSummaryCopyMapping[userState].title}</h1>
+			<p css={descriptions}>
+				{onboardingSummaryCopyMapping[userState].description}
+			</p>
+
+			<Stack
+				space={0}
+				cssOverrides={css`
+					margin-top: ${space[5]}px;
+				`}
+			>
+				<Button
+					priority="primary"
+					cssOverrides={buttonOverrides}
+					onClick={() => handleStepNavigation(OnboardingSteps.GuardianApp)}
+				>
+					Explore your benefits
+				</Button>
+				<Button
+					priority="subdued"
+					cssOverrides={buttonOverrides}
+					onClick={() => handleStepNavigation(OnboardingSteps.Completed)}
+				>
+					Skip
+				</Button>
+			</Stack>
+
+			<div css={newsletterContainer}>
+				<Stack space={1}>
+					<h2 css={boldDescriptions}>Saturday Edition newsletter</h2>
+					<p css={descriptions}>
+						An exclusive email highlighting the week’s best Guardian journalism
+						from our editor-in-chief, Katharine Viner
+					</p>
+				</Stack>
+				<ToggleSwitch
+					checked={switchNewsletterEnabled}
+					onClick={() => setSwitchNewsletterEnabled(!switchNewsletterEnabled)}
+				/>
+			</div>
+		</Stack>
+	);
+}
+
+function OnboardingSummary({
+	productKey,
+	landingPageSettings,
+	supportRegionId,
+	payment,
+	ratePlanKey,
+}: OnboardingProps) {
+	const order = getThankYouOrder();
+	const productSettings =
+		productKey && landingPageSettings.products[productKey];
+	const { windowWidthIsLessThan } = useWindowWidth();
+
+	const { currencyKey } = getSupportRegionIdConfig(supportRegionId);
+	const amountPaidToday = simpleFormatAmount(
+		getCurrencyInfo(currencyKey),
+		payment.finalAmount,
+	);
+
+	const billingPeriod = ratePlanKey
+		? ratePlanToBillingPeriod(ratePlanKey)
+		: undefined;
+	const periodNoun = billingPeriod ? getBillingPeriodNoun(billingPeriod) : '';
+	const fullAmount = `${amountPaidToday}/${periodNoun}`;
+
+	const nextPaymentDate = new Date();
+	if (billingPeriod === BillingPeriod.Monthly) {
+		nextPaymentDate.setMonth(nextPaymentDate.getMonth() + 1);
+	} else if (billingPeriod === BillingPeriod.Annual) {
+		nextPaymentDate.setFullYear(nextPaymentDate.getFullYear() + 1);
+	} else if (billingPeriod === BillingPeriod.Quarterly) {
+		nextPaymentDate.setMonth(nextPaymentDate.getMonth() + 3);
+	}
```

Comment:
if there's another billing period we would just say the next payment date is today?  should we change to effectively say `else 'unknown'` rather than putting an incorrect date?